workflow-steps/install-node/main.js

```diff
@@ -0,0 +1,36 @@
+const { execSync } = require('child_process');
+const { existsSync } = require('fs');
+
+// TODO: switch step to remove nvm instaall when nvm is in the base image
+if (process.env.NX_CLOUD_NODE_VERSION || existsSync('.nvmrc')) {
+  try {
+    // have to run as single command to keep the nvm env vars in the same shell
```

Comment:
NOTE: there was some issues when I tried running these commands as discrete commands where I'd get permission denied and other errors when trying to load in the nvm.sh file, so I just switch to mimic the equivalent in bash scripts to get things working. 

I also tried adding nvm into the base image, but ran into the same issues with permission errors and unable to find things correctly in PATH. unsure what the exact issue is there.